frontend/src/metabase/components/EmptyState.css

```diff
@@ -0,0 +1,15 @@
+:local(.emptyState) {
+    composes: text-centered text-brand-light from "style";
```

Comment:
Makes sense. I thought I might have needed to add some custom CSS for some of these but in a lot of cases it turns out the existing classes were enough. I'll do a sweep and convert those into inline classnames.